src/components/presentations/NewArticle/NewArticle.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import NewArticleForm from './NewArticleForm/NewArticleForm';
+
+class CreateArticle extends Component {
```

Comment:
The name of this class should be `NewArticle` since the file's name is `NewArticle.jsx` is there any reason this component has a different name? @AnayoOleru

Reply:
@rotimi-babalola No reason at all, I'll fix that